### PR TITLE
Nit fixes on README.md

### PR DESCRIPTION
**Pull request by cemozerr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,18 +11,17 @@
   </a>
 </div>
 
-## What is Sovereign?
+## What is Sovereign SDK?
```
Comment (preston-evans98) [inline README.md:14]: If we're going to say "Sovereign SDK", it needs to have a "the" in front of it.

File: README.md [hunk 2]
```diff
@@ -11,18 +11,17 @@
...
 
-Sovereign is a free and open-source toolkit for building zk-rollups **that is currently under development**. Sovereign consists of three
-logical components.
+Sovereign SDK is a free and open-source toolkit for building zk-rollups **that is currently under development**. Sovereign SDK consists of three logical components.
```
Comment (preston-evans98) [inline README.md:16]: Missing the "the" again :)

File: README.md [hunk 3]
```diff
@@ -11,18 +11,17 @@
...
 
 1. Sovereign Core, which defines a minimal interface for zk-rollups
 1. The Sovereign Module System, an opinionated framework for building rollups with Sovereign Core
 1. The Sovereign Full Node, a client implementation capable of running any rollup which implements the interfaces defined by Sovereign Core
 
 ### Sovereign Core: the rollup interface
 
-At the heart of Sovereign is [Sovereign Core](./core/specs/overview.md), which defines the _interfaces_ that rollups
+At the heart of Sovereign SDK is [Sovereign Core](./core/specs/overview.md), which defines the _interfaces_ that rollups
```
Comment (preston-evans98) [inline README.md:24]: Missing "the"

File: README.md (hunk 1 above)
Comment (cemozerr) [inline README.md:14]:
```suggestion
## What is the Sovereign SDK?
```

File: README.md (hunk 2 above)
Comment (cemozerr) [inline README.md:16]:
```suggestion
The Sovereign SDK is a free and open-source toolkit for building zk-rollups **that is currently under development**. Sovereign SDK consists of three logical components.
```

File: README.md (hunk 3 above)
Comment (cemozerr) [inline README.md:24]:
```suggestion
At the heart of the Sovereign SDK is [Sovereign Core](./core/specs/overview.md), which defines the _interfaces_ that rollups
```

**Review by preston-evans98 — APPROVED** [review]
This change looks fine, but will be outdated once we resolve the naming question around Sovereign Core. Still, I'm fine with merging as-is.

**Comment by cemozerr** [comment]
Closing in favor of: https://github.com/Sovereign-Labs/sovereign/pull/313

**Pull request closed without merging**